src/libs/actions/Report.ts

```diff
@@ -842,12 +842,29 @@ function addActions(reportID: string, notifyReportID: string, timezoneParam: Tim
     notifyNewAction(notifyReportID, lastAction?.actorAccountID, lastAction);
 }
 
-/** Add an attachment and optional comment. */
-function addAttachment(reportID: string, notifyReportID: string, file: FileObject, timezoneParam: Timezone, text = '', shouldPlaySound?: boolean) {
```

Comment:
Can you clarify why we removed shouldPlaySound? Is it safe to do so? I think we should avoid changing behavior that isn’t part of the issue’s scope.